packages/vscode-docker-registries/src/clients/GenericRegistryV2/GenericRegistryV2WizardPromptStep.ts

```diff
@@ -42,6 +43,10 @@ export class GenericRegistryV2WizardPromptStep<T extends GenericRegistryV2Wizard
             const uri = Uri.parse(value);
             scheme = uri.scheme;
             authority = uri.authority;
+
+            if (wizardContext.connectedRegistries.includes(uri.toString().toLowerCase())) {
```

Comment:
Are the values in `connectedRegistries` already forced to lowercase?

Reply:
yeah, we do it before we store it https://github.com/microsoft/vscode-docker-extensibility/blob/8989b049733fc924557679b1eb7de53e3d3ffc75/packages/vscode-docker-registries/src/clients/GenericRegistryV2/GenericRegistryV2DataProvider.ts#L105